.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,44 @@
-## Description
+**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be compeleted as written. Provide known gaps to testing that may raise the risk of merging to production.
 
 
-## Original issue(s)
-department-of-veterans-affairs/va.gov-team#0000
+## Summary
 
+- *(Summarize the changes that have been made to the platform)*
```

Comment:
These bullet points should probably be collapsed to a single line, if possible. Actually, I wonder if none of these sections should have a description, and rather we should have a separate document describe the intention for each section.  

The way I currently use our PR template, is I go through each section and delete the comments within, something I've had to do hundreds of time. It would be nice if the actual PR template was set up to expedite PR creation, and other documentation used to train around the process (if we have a separate document then I think all these bullet points are probably relevant)

Reply:
tracking what your asking. I'd like to keep the template more verbose for the time being and if we can get enough participation then lets work on auto deleting